support-frontend/app/controllers/StripeController.scala

```diff
@@ -72,6 +72,20 @@ class StripeController(
     }
   }
 
+  def createSetupIntentPRB: Action[SetupIntentRequest] = PrivateAction.async(circe.json[SetupIntentRequest]) { implicit request =>
+
+    val cloudwatchEvent = createSetupIntentRequest(stage, "CSRF-only")
+    AwsCloudWatchMetricPut(client)(cloudwatchEvent)
+
+    stripeService(request.body.stripePublicKey).fold(
+      error => {
+        logger.error(s"Returning status InternalServerError for Create Stripe Intent request because: $error")
```

Comment:
`logger.error(s"Returning status InternalServerError for Create Stripe Intent PRB request because: $error")`